src/supermarket/engines/fieri/app/models/cookbook_artifact.rb

```diff
@@ -97,8 +85,8 @@ def cleanup
   # @return [Tempfile] the artifact
   #
   def download
-    File.open(Tempfile.new("archive"), "wb") do |saved_file|
-      URI.open(url, "rb") do |read_file|
+    File.open(Tempfile.new('archive'), 'wb') do |saved_file|
```

Comment:
There's a few of these quote changes around that will make this code fail Chefstyle linting now

Reply:
Hi, I also noticed the same.. let me take look thanks